pylint/checkers/refactoring/refactoring_checker.py

```diff
@@ -1094,13 +1100,15 @@ def _check_super_with_arguments(self, node):
             or not isinstance(node.args[0], nodes.Name)
             or not isinstance(node.args[1], nodes.Name)
             or node_frame_class(node) is None
-            or node.args[0].name != node_frame_class(node).name
+            or node.args[0].name != node_frame_class(node).name  # type: ignore[union-attr]
```

Comment:
```suggestion
            # TODO: PY38: Use walrus operator, this will also fix the mypy issue
            or node.args[0].name != node_frame_class(node).name  # type: ignore[union-attr]
```